.github/workflows/devcontainer.yml

```
@@ -0,0 +1,47 @@
+      - ".devcontainer/**"
+      - ".github/workflows/_devcontainer.yml"
+      - ".github/workflows/devcontainer.yml"
+jobs:
+  configuration:
+    runs-on: ubuntu-latest
+    outputs:
+      image_name: ${{ steps.set_image_name.outputs.image_name }}
+    steps:
+      - name: Set Image Name
+        id: set_image_name
+        run: |
```

actionlint (1.7.9)

shellcheck reported issue in this script: SC2086:info:6:49: Double quote to prevent globbing and word splitting

(shellcheck)